src/framework/lib/genesisframework/task.rb

```diff
@@ -112,32 +112,55 @@ def install provider, *what
               raise 'yum install exited with status: ' + $?.exitstatus.to_s
             end
           elsif provider == :gem
+            # convert arguments in "what" to a gem_name => [requires_list] structure
+            gems = {}
+            what.each { |item|
+              if item.is_a?(Hash)
+                gems.merge! item
+              else
+                gems[item] = []
```

Comment:
I think it is slightly better to do:
gem[item] = [item]
instead to keep all the complexity around standardizing the given to gem_name... structure local to this block.
Not a big deal, but a slight win IMO.